template.yaml

```diff
@@ -337,21 +366,50 @@ Resources:
             - Stack ${StackTagName} Environment ${EnvironmentTagName} EFS ${ResourceName} VPC Security Group
             - ResourceName: ghostAPI
       VpcId: !Ref DefaultVPCId
-  ghostLBSecurityGroup:
+  ghostGWSecurityGroup:
     Type: AWS::EC2::SecurityGroup
     Properties:
-      GroupDescription: Load Balancer Security Group
-      SecurityGroupIngress:
-        - CidrIp: 0.0.0.0/0
-          FromPort: 80
-          IpProtocol: tcp
-          ToPort: 80
+      GroupDescription: HTTP Gateway Security Group
       Tags:
         - Key: Name
           Value: !Sub
             - Stack ${StackTagName} Environment ${EnvironmentTagName} API ${ResourceName} VPC Security Group
             - ResourceName: ghostLB
-      VpcId: !Ref DefaultVPCId            
+      VpcId: !Ref DefaultVPCId
+  ghostProxy:
+    Type: AWS::Serverless::HttpApi
+    Properties:
+      DefinitionBody:
+        openapi: '3.0'
+        info:
+          title: !Sub ${AWS::StackName}-HttpApi
+          version: '1.0'
+        paths:
+          /ghost:
```

Comment:
I'm still unsure why this doesn't break the /ghost route. Maybe because it doesn't have any integration specified it's ignored when the API is provisioned? My IaC purist gut says to remove it, but I understand you may want it there to be illustrative.

Alternatively, you could copy/paste the same integration block for the /{proxy+} route. It would be duplicative, but it would be "correct", and it would call out the separate path for illustrative purposes.

Reply:
👍